velox/dwio/parquet/tests/duckdb_reader/ParquetTableScanTest.cpp

```diff
@@ -227,6 +252,51 @@ TEST_F(ParquetTableScanTest, countStar) {
   assertQuery(plan, {split}, "SELECT 20");
 }
 
+class NativeParquetTableScanTest
```

Comment:
This should be in its own file in `tests/reader`

Reply:
Updated, thanks for the suggestion.